tianshou/trainer/offline.py

```diff
@@ -75,12 +77,13 @@ def offline_trainer(
     start_time = time.time()
     test_collector.reset_stat()
 
-    test_result = test_episode(
-        policy, test_collector, test_fn, start_epoch, episode_per_test, logger,
-        gradient_step, reward_metric
-    )
-    best_epoch = start_epoch
-    best_reward, best_reward_std = test_result["rew"], test_result["rew_std"]
+    if test_in_train:
```

Comment:
I don't think this is the same logic of `test_in_train` compared with other trainers. It is used for testing performance in the middle of an epoch.

Reply:
I changed the name to `disable_test`. I just need a switch to turn off testing in offline setting.

Reply:
So why not use `test_collector=None` to indicate?